packages/authorization/package.json

```diff
@@ -27,6 +27,7 @@
   "devDependencies": {
     "@loopback/build": "^2.0.3",
     "@loopback/testlab": "^1.6.3",
+    "@loopback/authentication": "^2.1.8",
```

Comment:
Please add some comments about this temporary workaround.

Reply:
@raymondfeng I added an entry in the README.md file to explain the user injection.

Reply:
BTW, let's change the initial version to be `0.0.1` so that the first release will be `0.1.0`.